waveform_editor/gui/io/filebrowser.py

```diff
@@ -147,7 +147,7 @@ def _update_files(
             paths.insert(0, "..")
             abbreviated.insert(0, "⬆ ..")
 
-        options = dict(zip(abbreviated, paths))
+        options = dict(zip(abbreviated, paths, strict=False))
```

Comment:
```suggestion
        options = dict(zip(abbreviated, paths, strict=True))
```

`abbreviated` and `paths` are guaranteed to have the same length (with above logic) `strict=True` captures this